File: Data Structures & Algorithms/longest-consecutive-sequence/submission-3.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums_seq = []

        for num in nums:
            if((num-1) not in nums):
                seq_to_append = 1
                curr_num = num
                while(True):
                    if((curr_num + 1) in nums):
                        curr_num += 1
                        seq_to_append += 1
                    else:
                        nums_seq.append(seq_to_append)
                        break
            else:
                continue


        return max(nums_seq, default=0)

File: Data Structures & Algorithms/maximum-subarray/submission-3.cs
public class Solution {
    public int MaxSubArray(int[] nums) {
        int res = nums[0];
        int currSum = nums[0];

        for (int i = 1; i < nums.Length; i++) {
            if (nums[i] < (currSum + nums[i])) {
                currSum += nums[i];
            }
            else {
                currSum = nums[i];
            }
            
            res = Math.Max(res, currSum);
        }

        return res;
    }
}
